da/common.py

```diff
@@ -37,21 +37,43 @@ def transposed(self) -> Self:
 BLSPublickey = bytes
 BLSPrivateKey = int
 BLSSignature = bytes
+BLSPopProof = BLSSignature
 
 
-class Bitfield(List[bool]):
+class KeyPossessionProofs(List[Optional[BLSPopProof]]):
     pass
 
 
 @dataclass
 class Attestation:
-    signature: BLSSignature
+    # Signature over the attestation message H(AggC, RowC...)
+    message_signature: BLSSignature
+    # Proof of possession on bls pop sk
+    proof_of_possession: BLSSignature
```

Comment:
@megonen Proof of Possession should go with the registration of your public key, right? We don't need to proof possession with each attestation?

mentioning because I believe putting PoP in attestation/certificate will blow up the certificate size

Reply:
So, thing goes like this:
* Upon registering the pk nodes also need to provide the pop 
* We keep them *both* available
* pop is only verified upon registration, meaning that we can really consider the pks trustful here already?

Reply:
I'll revert the latest commits then and just add an the pop in the initialization as a constrain.